### PR TITLE
Fix twig simple filter signature

### LINKED ISSUE #119: Twig Warning: Illegal offset type
Hello,

From the commit https://github.com/sonata-project/SonataFormatterBundle/commit/3c5c083202b2ff485890d9659ff8708b2276ded2 , Il have : An exception has been thrown during the compilation of a template ("Warning: Illegal offset type") in sandbox 2.4

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Fixes #119 

Thanks @wodka for reporting.

### CODE REVIEW

File: Twig/Extension/TextFormatterExtension.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ public function getTokenParsers()
     public function getFilters()
     {
         return array(
-            'format_text' => new \Twig_SimpleFilter($this, 'transform'),
+            new \Twig_SimpleFilter('format_text', [$this, 'transform']),
```
Comment (rande) [inline Twig/Extension/TextFormatterExtension.php]: need to be array, you know php5.3

Comment (Soullivaneuh) [inline Twig/Extension/TextFormatterExtension.php]: Oh yes, php 5.3...

**Comment by rande** [comment]
ok to test

**Comment by Soullivaneuh** [comment]
> SonataCi / Sandbox / PHPUnit — None

Well, when for details link? :-)

**Comment by rande** [comment]
@Soullivaneuh I think you have the credentials on jenkins. but here the errors: 1) Sonata\Bundle\QABundle\Tests\SonataPageRenderBlockCommandTest::testDump
Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Illegal offset type") in "SonataPageBundle:Block:block_container.html.twig".

**Comment by Soullivaneuh** [comment]
>  I think you have the credentials on jenkins

Don't know. But a public link to permits contributors to see errors is a must have IMHO.

**Comment by wodka** [comment]
@rande is there more of the stack trace? this looks like the same error but in a different file.

If I apply just the change It starts working again (I do not use the page bundle in my project)

**Comment by Tibase** [comment]
It's good for me with this fix.

**Comment by rande** [comment]
I will chek the PageBundle regression.

**Comment by Tibase** [comment]
Thanks

**Pull request merged**